src/features/commands/about.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
                                 .setStyle('LINK'),
                             new MessageButton()
                                 .setLabel('Get Dotsimus')
-                                .setURL('https://Dotsimus.com')
+                                .setURL('https://discord.com/oauth2/authorize?client_id=731190736996794420&permissions=67497175&redirect_uri=https%3A%2F%2Fdotsimus.com&response_type=code&scope=bot%20identify%20applications.commands')
```

Comment:
This is outside of poll feature scope too and this link needs to be updated to include thread creation, viewing, writing permissions in its scope as it is outdated.

Current perms which this link has are shown below
![image](https://user-images.githubusercontent.com/11031051/135342946-f372d6fb-ef1a-4579-9b3c-740b37211c94.png)